roles/openshift_provisioning_aws_vpc/molecule/default/tests/test_default.py

```diff
@@ -0,0 +1,21 @@
+# import boto
+def test_vpc_exists(Ansible):
```

Comment:
@cooktheryan since this is using an Ansible task and the ec2_vpc_net_facts module, I can verify that the vpc is configured the way I expect